cfme/tests/containers/test_rc_selector.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from cfme.containers.replicator import Replicator
+from utils import testgen, version
+from cfme.web_ui import CheckboxTable, toolbar as tb
+from utils.appliance.implementations.ui import navigate_to
+import random
+from itertools import chain
+from cfme.containers.provider import ContainersProvider
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda provider: version.current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(2)]
+pytest_generate_tests = testgen.generate(
+    [ContainersProvider], scope='function')
+
+
+@pytest.mark.polarion('CMP-9973')
+def test_rc_selector(provider):
+    """ This test verifies the data integrity for the
+        rc selector table
+    """
+
+    navigate_to(Replicator, 'All')
+    tb.select('List View')
+    list_selectors = provider.mgmt.list_replication_controller_selector()
+    list_selectors_values = list(chain.from_iterable(
+        [d.values() for d in list_selectors]))
+    list_tbl_rc = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+    cls_instances = [r.name.text for r in list_tbl_rc.rows()]
+    rand_smpl_list = [cls_instances[i] for i in
+                      sorted(random.sample(xrange(len(cls_instances)), 1))]
```

Comment:
Only 1 sample?, if so you don't need the for loop (I suggest 2 or 3 samples).
Also, you can just do `random.sample(cls_instances, <num_of_samples>) ` and you will get the same results.
Also, I'd suggest to use navigate_and_get_rows functions that located at cfme/containers/provider/.__init__.py, it'll reduce the code and more safe to use it, but first, you have to get latest and rebase (should be easy since you don't have conflicts)